src/Client.ts

```diff
@@ -284,7 +288,8 @@ export default class Client extends EventEmitter {
             jid: '',
             transports: {
                 bosh: true,
-                websocket: true
+                websocket: true,
+                tcp: false,
```

Comment:
TCP should also be enabled by default
```suggestion
                tcp: true,
```